src/components/GOODS_FROM_SERVER_API.ts

```diff
@@ -0,0 +1,13 @@
+import { Good } from '../types';
```

Comment:
it is not a component and should not be in the components folder
file should not be named in UPPERCASE